tasks/db_mysql.yml

```
@@ -12,7 +12,11 @@
   vars:
     nc_mysql_deps:
       - "php{{ php_ver }}-mysql"
-      - "python{{ '3' if ansible_python.version.major == 3 else '' }}-pymysql"
+      - "python3-pymysql"
+
+- name: Ensure MySQL is started and enabled on boot.
+  service: "name={{ mysql_daemon }} state=started enabled={{ nextcloud_db_enabled_on_startup }}"
```

You should move this definition to handlers and use this notation:
```suggestion
  service: 
    name={{ mysql_daemon }}
    state=started
    enabled={{ nextcloud_db_enabled_on_startup }}"
```